src/main/java/androidx/graphics/Bitmap.kt

```diff
@@ -111,3 +113,30 @@ inline fun createBitmap(
 ): Bitmap {
     return Bitmap.createBitmap(width, height, config, hasAlpha, colorSpace)
 }
+
+/**
+ * Creates a new bitmap, cropped from this bitmap. If the specified [x], [y],
+ * [width], [height] are the same as the current width and height of this bitmap,
+ * this bitmap is returned and no new bitmap is created.
+ *
+ * @param x The x coordinate of the first pixel.
+ * @param y The y coordinate of the first pixel.
+ * @param width The width.
+ * @param height The height.
+ * @return the cropped bitmap
+ */
+inline fun Bitmap.crop(x: Int, y: Int, width: Int, height: Int) =
+    Bitmap.createBitmap(this, x, y, width, height)
+
+/**
+ * Creates a new bitmap, rotated from this bitmap by [degrees] - the specified number of degrees,
+ * with a pivot point at ([px], [py]). The pivot point is the coordinate that should remain
```

Comment:
There's no pivot point in the API

Reply:
sorry for that, i added it.

Reply:
Thanks for the change. The pivot point should probably be set to width/2.0f and height/2.0f by default.

Reply:
I added the default pivot point.